docs/index.md

```diff
@@ -13,33 +13,52 @@ Getting Started
 
 ```{toctree}
 ---
-caption: Data Flow and Storage Management
+caption: Computational Resource Allocation and Processing
 maxdepth: 2
-hidden: true
+hidden: false
 ---
-Storage Levels and Data Flow
-Managing Storage with the Inductiva API
+Ivan_Computational_Resources
+Shared VM Pool Integration
+Machine Groups
+Setting up an MPI
```

Comment:
I think the `Ivan_Computational_Resources` should be at the end since it is more technical and in-depth.

Reply:
I think we do have a placeholder for the Shared Pool in the Machines.md 
[Task Queue and Shared VM Pool Integration]

Reply:
I am not finding that ReadMe and is not rendering it on the ReadTheDocs. Moreover, the Machine Groups name should change to Machines or the file to `Machine Groups.md`